vllm/model_executor/models/deepseek_v2.py

```diff
@@ -115,7 +115,7 @@ def __init__(
                                 hidden_size=config.hidden_size,
                                 intermediate_size=config.moe_intermediate_size,
                                 reduce_results=False,
-                                renormalize=config.norm_topk_prob,
+                                renormalize=True,
```

Comment:
Why is this changed from the config?

Reply:
Ah this was to fix the memory issue for awq. I realized this was the flag that was triggering the illegal memory issue so set it to True to prevent the error. Reverting